File: src/nomanssave/gC.java
package nomanssave;

public class gC {
  private final eY oG;
  
  public static gC y(eY parameY) {
    return new gC(parameY);
  }
  
  private gC(eY parameY) {
    this.oG = parameY;
  }
  
  public int dV() {
    return this.oG.J("PrimaryShip");
  }
  
  public void aG(int paramInt) {
    this.oG.b("PrimaryShip", Integer.valueOf(paramInt));
  }
  
  public int dM() {
    return this.oG.J("ShipHealth");
  }
  
  public void aB(int paramInt) {
    this.oG.b("ShipHealth", new Integer(paramInt));
  }
  
  public int dN() {
    return this.oG.J("ShipShield");
  }
  
  public void aC(int paramInt) {
    this.oG.b("ShipShield", new Integer(paramInt));
  }
}


/* Location:              E:\NMSSE_Source\NMSSaveEditor.jar!\nomanssave\gC.class
 * Java compiler version: 8 (52.0)
 * JD-Core Version:       1.1.3
 */

File: NMSSaveEditor/nomanssave/mixed/gC.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{
public class gC {
   private eY oI;

   public static gC y(eY var0) {
      return new gC(var0);
   }

   private gC(eY var1) {
      this.oI = var1;
   }

   public int dV() {
      return this.oI.J("PrimaryShip");
   }

   public void aG(int var1) {
      this.oI.b("PrimaryShip", (Object)var1);
   }

   public int dM() {
      return this.oI.J("ShipHealth");
   }

   public void aB(int var1) {
      this.oI.b("ShipHealth", (Object)(new Integer(var1)));
   }

   public int dN() {
      return this.oI.J("ShipShield");
   }

   public void aC(int var1) {
      this.oI.b("ShipShield", (Object)(new Integer(var1)));
   }
}

}
